client/python/samples/resnet_grpc_predict.py

```diff
@@ -0,0 +1,119 @@
+#
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+import numpy as np
+import classes
+from ovmsclient import make_grpc_client, make_grpc_predict_request
+
+parser = argparse.ArgumentParser(description='Make prediction using images in numpy format')
+parser.add_argument('--images_numpy_path', required=False, default='./utils/imgs.npy',
+                    help='path to image')
+parser.add_argument('--labels_numpy_path', required=False,
+                    help='numpy in shape [n,1] - can be used to check model accuracy')
```

Comment:
Can we drop using `.npy` files and replace those parameters with `--images_dir` that would contain path to the directory with images in JPG format? 
Also let's drop passing labels. The script would not calculate the accuracy and just print the predicted label for each JPG file what do you think?